deploy/template/usr/local/bin/start-docker-worker

```diff
@@ -1,6 +1,3 @@
 #!/bin/bash
 
-DOCKER_WORKER='node /home/ubuntu/docker_worker/src/bin/worker.js'
-DOCKER_WORKER_OPTS=
-
-$DOCKER_WORKER $DOCKER_WORKER_OPTS --host $HOST production 2>&1 | logger --tag docker-worker
+/usr/local/bin/start-worker /etc/start-worker.yml 2>&1 | logger --tag docker-worker
```

Comment:
Instead of changing the existing script, could you please create another script? Since it is not production ready yet, it may impact deployments in the mean time.

Reply:
Oh, yes, that's silly of me!  Maybe I'll keep this alive on a branch for the moment -- the alternative would be to update the packer build process to output another two sets of AMIs.  I'll also focus on getting feature parity here.